### ISSUE TITLE
Interop with Kwik/Flupke fails

### ISSUE THREAD

**Issue by WesleyRosenblum** [hyperium/h3#118]
s2n-quic received an [issue](https://github.com/aws/s2n-quic/issues/1461) pointing out that the QUIC interop HTTP/3 test between s2n-quic server and Kwik client fails. s2n-quic server receives HTTP/3 requests for 3 files, but hyperium/h3 does not request s2n-quic server to send the files.

We also noticed that the same interop test fails between Quinn server and Kwik client.

Since both of these servers are using hyperium/h3, and I couldn't find an issue with the s2n-quic server implementation, there might be a bug in hyperium/h3.

Packet captures and logs are available here:
https://interop.seemann.io/logs/2022-08-18T08:08/s2n-quic_kwik/http3/

Thanks!

**Comment by seanmonstar** [comment]
That's interesting! The link to the logs seems blank, I'm wondering if they expired? Anywho, I wonder what Kwik does differently that `h3` doesn't like... Since I assume the same set of requests work fine with other clients.

**Comment by WesleyRosenblum** [comment]
Yeah looks like they remove the old runs, you can view the latest runs here: https://interop.seemann.io/?test=3&client=kwik, just click the red "3" to see the logs and packet captures

**Comment by ptrd** [comment]
> That's interesting! The link to the logs seems blank, I'm wondering if they expired? Anywho, I wonder what Kwik does differently that `h3` doesn't like... Since I assume the same set of requests work fine with other clients.

It works with some, it fails with some. Currently, it fails with 3 other clients and all 4 have successful h3 interactions with all servers (see https://interop.seemann.io/?test=3 )
